README.md

```diff
@@ -1,9 +1,16 @@
+<img align="right" width="200px" src="https://github.com/FluxML/NNlib.jl/raw/master/docs/src/assets/logo.png">
+
 # NNlib.jl
 
+[![Documentation][docs-dev-img]][docs-dev-url]
 [![CI](https://github.com/FluxML/NNlib.jl/actions/workflows/ci.yml/badge.svg)](https://github.com/FluxML/NNlib.jl/actions/workflows/ci.yml)
 [![Coverage](https://codecov.io/gh/FluxML/NNlib.jl/branch/master/graph/badge.svg)](https://codecov.io/gh/FluxML/NNlib.jl) 
-[![Documentation](https://img.shields.io/badge/docs-fluxml.ai-blue)](https://fluxml.ai/Flux.jl/stable/models/nnlib/)
 
+[docs-stable-img]: https://img.shields.io/badge/docs-stable-blue.svg
+[docs-stable-url]: https://fluxml.ai/NNlib.jl/stable/
```

Comment:
Will this work once we tag a new version?

Reply:
Oh I don't know, I copied from Optimisers where it also doesn't work.

Reply:
Worth a try then I guess.